scripts/README.md

```diff
@@ -27,6 +27,7 @@ The following steps must be proposed as a pull request.
 - [ ] Clean the generated changelog, to match the format of [https://github.com/mui/mui-x/releases](https://github.com/mui/mui-x/releases).
 - [ ] Update the root `package.json`'s version
 - [ ] Update the versions of the other `package.json` files and of the dependencies with `yarn release:version`.
+- [ ] Fix manually the package version in `x-date-picker/package.json` and `x-date-picker-pro/package.json`.
```

Comment:
Note that in Material UI we had to set the `version` to independent as we had too many npm packages in the alpha release state. https://github.com/mui/material-ui/blob/88ad7d4e623183b8e20702e7961ef9c8de330dc7/lerna.json#L4.

If the date picker becomes stable soon enough and we don't have new unstable npm packages, maybe we can stick to the current configuration of MUI X. Hopefully, in Material UI, follow the same direction as well.